cabal.project

```diff
@@ -1,13 +1,14 @@
 packages: .
 
 -- these upper bounds are required in order to not break payload validation in chainweb
-constraints: base16-bytestring <1
+constraints: base16-bytestring <1.1
 constraints: prettyprinter <1.6.1
-constraints: base64-bytestring <1.1
```

Comment:
These were capped due to replay errors (the error message schema changed between 1.0 and <1).